packages/mui-material/src/styles/ThemeProviderWithVars.tsx

```diff
@@ -2,20 +2,18 @@
 import * as React from 'react';
 import { unstable_createCssVarsProvider as createCssVarsProvider, SxProps } from '@mui/system';
 import styleFunctionSx from '@mui/system/styleFunctionSx';
-import extendTheme, { SupportedColorScheme, CssVarsTheme } from './extendTheme';
+import createThemeWithVars, { SupportedColorScheme, CssVarsTheme } from './createThemeWithVars';
```

Comment:
We should use the public API as much as possible, e.g. `createTheme({ cssVariables: true })`, as this is how users would use it.